docs/UserGuide.adoc

```diff
@@ -54,66 +54,62 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 * Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 ====
 
-=== Viewing help : `help`
-
-Format: `help`
-
-
-=== Clearing all entries : `avail`
-
-Lists all the available rooms of that type for the next NUM_DAYS days +
-Format: `avail r/ROOM TYPE num/NUM_DAYS`
-
-Lists all available rooms from start to end date. Dates have to be given in YYMMDD format
-
-=== Adding a guest: `add`
+=== Adding a person: `add`
 
-Adds a guest to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a guest to the guest list for the purpose of assigning a room +
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG] r/ROOM_NUMBER from/START_DATE to/END_DATE`
 
 [TIP]
 A guest can have any number of tags (including 0)
 
-
 Examples:
 
-* `avail r/Double Room num/5`
-* `avail r/Double Room ds/20180910 de/21080912`
+* `add n/John Smith p/98765432 e/johnsmith@gmail.com t/VIP r/305 from/09/12/18 to/10/12/18`
+Add a guest "John Smith" to room 305 for a period of stay from 09/12/18 to 10/12/18.
 
-=== Listing all guests : `list`
+=== Adding a person: `addx`
 
+Adds a guest to the guest list NOT for the purpose of assigning a room (perhaps add to archived guest list) +
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]...`
 
-Shows a list of all room types and cost of room +
-Format: `list`
+Examples:
 
-=== Adding a person: `add`
+* `add n/guest g/S9876543B`
 
-Adds a guest to the guest list for the purpose of assigning a room +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]...`
+=== Checkout room: `checkout`
 
-=== Adding a person: `addx`
+Checks out the room and the guest staying in it. The guest is deleted from the guest list and the room is then marked
+for housekeeping. +
+Format: `checkout ROOM_NUMBER`
 
-Adds a guest to the guest list NOT for the purpose of assigning a room +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]...`
+Examples:
 
-[TIP]
-A guest can have any number of tags (including 0)
+* `checkout 305` +
+Checks out room 305 and the guest staying in it.
```

Comment:
I thought our range of rooms was 001-100.

Reply:
Korrect, my bad. Updated in latest commit.